arbnode/batch_poster.go

```diff
@@ -506,23 +506,28 @@ func (b *BatchPoster) addEspressoBlockMerkleProof(
 			return fmt.Errorf("this msg has not been included in hotshot")
 		}
 
-		snapshot, err := b.lightClientReader.FetchMerkleRoot(jst.Header.Height, nil)
+		_, err = b.lightClientReader.FetchMerkleRoot(jst.Header.Height, nil)
```

Comment:
Do we need to fetch the merkle root now?

Reply:
It seems that the only thing we needed from the snapshot was the hotshot height. I think were fine to remove this now.

Reply:
Why do we still need this call at all?